.github/workflows/on-dev-merge.yaml

```diff
@@ -3,8 +3,15 @@ on:
   push:
     branches:
       - dev
+
+permissions:
+  contents: read
+
 jobs:
   notify-team:
+    permissions:
+      contents: read
+      pull-requests: read
```

Comment:
I would suggest to either move the pull request permission to the top level or remove the top level permissions.

Reply:
Done!